src/engine/client/key_identification.cpp

```diff
@@ -186,6 +186,32 @@ Key StringToKey(Str::StringRef str)
     return Key::NONE;
 }
 
+std::string KeyToStringUnprefixed( Key key ) {
```

Comment:
I don't understand, what is the point of forming a string with words like `ESCAPE`, `0x63` etc.?